library/gprelations.inc.php

```diff
@@ -7,23 +7,19 @@
 // of the License, or (at your option) any later version.
 //
 // This module supports use of the gprelations table to maintain
-// many-to-many relationships (linkings) among the following other
-// tables.  For each, a corresponding type code is assigned:
-//
-//  1 documents
-//  2 form_encounter (visits)
-//  3 immunizations
-//  4 lists (issues)
-//  5 openemr_postcalendar_events (appointments)
-//  6 pnotes
-//  7 prescriptions
-//  8 transactions (e.g. referrals)
+// many-to-many relationships (linkings) among database tables listed in
+// function getGpRelKey which supports use of literals for readability.
 //
 // By convention we require that type1 must be less than or equal to type2.
 //
-// As of this writing (2009-11-11), only documents-to-pnotes relations are
-// used. However expansion is anticipated, as well as the opportunity to
-// obsolete the issue_encounter table.
+// Since expansion is anticipated, as well as the opportunity to obsolete the 
+// issue_encounter table, please document linkings used in standard codebase.
+
+function getGpRelKey ($strTab) {
+    $rs = sqlQuery('SELECT seq FROM list_options WHERE list_id=? AND option_id=? AND activity=?',
```

Comment:
list_options.seq is only supposed to serve a cosmetic purpose, so the user can order the list in whatever sequence is most useful. You instead want to use title and/or option_value for associated values.